src/commands/bhInfo.js

```diff
@@ -73,8 +73,11 @@ async function feedEmbed(item) {
       .setDescription(
         (content ? `${content} ` : "") + `[[Read More]](${item.link[0]})`
       )
-      .setTimestamp(date)
-      .setURL(item.link[0]);
+      .setTimestamp(date);
+    if (item.link[0]) {
+      embed.setURL(item.link[0]);
+    }
```

Comment:
Caused an issue when there was no link, such as when I didn't set a URL for the home page